src/client/conn.rs

```diff
@@ -485,6 +491,32 @@ impl Builder {
         self
     }
 
+    /// Sets the [`SETTINGS_INITIAL_WINDOW_SIZE`][spec] option for HTTP2
+    /// stream-level flow control.
+    ///
+    /// Default is 65,535
+    ///
+    /// [spec]: https://http2.github.io/http2-spec/#SETTINGS_INITIAL_WINDOW_SIZE
+    pub fn http2_initial_stream_window_size(&mut self, sz: impl Into<Option<u32>>) -> &mut Self {
+        if let Some(sz) = sz.into() {
+            self.h2_builder.initial_window_size(sz);
+        }
+        self
+    }
+
+    /// Sets the [`SETTINGS_INITIAL_WINDOW_SIZE`][spec] option for HTTP2
```

Comment:
This is more of a bit, but there isn't actually an HTTP2 setting for the initial *connection* window size. It *always* starts at 65,535, and this is set, we just immediately send  a `WINDOW_UPDATE` for the connection to make up the difference. So I'd just remove the setting reference, and just mention this changes the max connection flow control.

Reply:
Okay that makes sense. I was back and forth on documentation here because of that situation, but glad to mention the difference here instead.